src/cljs/athens/hotkeys.cljs

```diff
@@ -0,0 +1,75 @@
+(ns athens.hotkeys
+  (:require
+    ["react-hotkeys" :refer [GlobalHotKeys HotKeys configure]]
+    [re-frame.core :refer [dispatch subscribe]]
+    [athens.util :as util]
+    [athens.router :as router]))
+
+(configure #js{:ignoreTags #js[]})
+
+(def modkey
+  (let [os (util/get-os)]
+    (or (and (= os :mac) "meta")
+      (and (= os :windows) "ctrl")
+      (and (= os :linux) "ctrl"))))
+
+(defn with-mod
+  [combination]
+  (str modkey "+" combination))
+
+(defn not-editing
+  []
+  (nil? @(subscribe [:editing/uid])))
```

Comment:
This kind of `deref` in helper fn worries me a bit, because we're loosing reactive nature of subscription.
Also Clojure convention for naming predicates is to end them with `?` so in this case `not-editing?`, this way we all know when it's used it returns boolean.
Here you could either wrap body in `reagent.ratom/reaction` or create dedicated subscription that does `nil?` check.

Reply:
Sounds good. Here is kinda safe because it is called inside an event handler (so it gets the up to date value on each call)